libs/blocks/merch/merch.js

```diff
@@ -198,22 +198,47 @@ export async function fetchCheckoutLinkConfigs(base = '') {
   return fetchCheckoutLinkConfigs.promise;
 }
 
-export async function getCheckoutLinkConfig(productFamily) {
+/**
+ * @param {*} productFamily product family code
```

Comment:
Do we needs the docs?

Reply:
not really. I guess you want to save bytes :)